infra/cifuzz/cifuzz.py

```diff
@@ -109,34 +122,35 @@ def build_fuzzers(project_name,
 
   logging.info("Using %s sanitizer.", sanitizer)
 
-  git_workspace = os.path.join(workspace, 'storage')
-  os.makedirs(git_workspace, exist_ok=True)
   out_dir = os.path.join(workspace, 'out')
   os.makedirs(out_dir, exist_ok=True)
 
-  # Detect repo information.
-  inferred_url, oss_fuzz_repo_path = build_specified_commit.detect_main_repo(
-      project_name, repo_name=project_repo_name)
-  if not inferred_url or not oss_fuzz_repo_path:
+  # Build Fuzzers using docker run.
+  inferred_url, project_builder_repo_path = (
+      build_specified_commit.detect_main_repo(project_name,
+                                              repo_name=project_repo_name))
+  if not inferred_url or not project_builder_repo_path:
     logging.error('Could not detect repo from project %s.', project_name)
     return False
-  src_in_docker = os.path.dirname(oss_fuzz_repo_path)
-  oss_fuzz_repo_name = os.path.basename(oss_fuzz_repo_path)
+  project_repo_name = os.path.basename(project_builder_repo_path)
+  src_in_project_builder = os.path.dirname(project_builder_repo_path)
+
+  manual_src_checkout_path = os.getenv('MANUAL_SRC_PATH')
```

Comment:
how about LOCAL_SRC_PATH, manual feels weird in env var and variable name.

Reply:
I don't agree LOCAL describes what we are doing better than MANUAL. Instead of automatically getting the source, CIFuzz uses one that is manual gotten by the caller and passed to CIFuzz.